File: Hashtable-JavaFX/src/main/java/com/example/demo2/model/Table.java
package com.example.demo2.model;

public class Table {
    public Table() {
    }

    private Node[] nodes;

    public Node[] getNodes() {
        return nodes;
    }

    public void setNodes(Node[] nodes) {
        this.nodes = nodes;
    }

    public Table(int size) {
        nodes  = new Node[size];
    }
}

File: Hashtable_Angular/src/app/dto/Table.ts
import {Node} from "./Node";

export class Table{
  private _nodes: (Node | null)[] = [];
  constructor(size : number) {
    this._nodes = new Array(size).fill(null);
  }

  get nodes(): (Node | null)[] {
    return this._nodes;
  }

  set nodes(value: Node[]) {
    this._nodes = value;
  }
}
